rules/TypeDeclaration/Rector/ClassMethod/ReturnTypeFromStrictScalarReturnExprRector.php

```diff
@@ -36,35 +45,48 @@ public function __construct(
     public function getRuleDefinition(): RuleDefinition
     {
         return new RuleDefinition('Change return type based on strict scalar returns - string, int, float or bool', [
-            new CodeSample(
+            new ConfiguredCodeSample(
                 <<<'CODE_SAMPLE'
 final class SomeClass
 {
-    public function run($value)
+    public function foo($value)
     {
         if ($value) {
             return 'yes';
         }
 
         return 'no';
     }
+
+    public function bar(string $value)
+    {
+        return strlen($value);
+    }
 }
 CODE_SAMPLE
-
                 ,
                 <<<'CODE_SAMPLE'
 final class SomeClass
 {
-    public function run($value): string
+    public function foo($value): string
     {
         if ($value) {
             return 'yes';
         }
 
         return 'no';
     }
+
+    public function bar(string $value): int
+    {
+        return strlen($value);
+    }
 }
 CODE_SAMPLE
+                ,
+                [
+                    ReturnTypeFromStrictScalarReturnExprRector::HARD_CODED_ONLY => true,
```

Comment:
This need false per default value for consistency

Reply:
Yip, and the example output was wrong as well 🙈 Noticed that myself too when once more going over your comments